tests/link/pytorch/test_elemwise.py

```diff
@@ -58,8 +58,9 @@ def test_pytorch_elemwise():
     compare_pytorch_and_py(fg, [[0.9, 0.9]])
 
 
+@pytest.mark.parametrize("fn", [ptm.sum, ptm.prod, ptm.max, ptm.min])
 @pytest.mark.parametrize("axis", [0, 1, (0, 1), (1, 2), (1, -1)])
```

Comment:
I think this is sufficient
```suggestion
@pytest.mark.parametrize("axis", [None, 0, 1, (0, -1)])
```